test/jdk/java/lang/Math/PowTests.java

```diff
@@ -57,6 +57,21 @@ static int testNonstrictPowCase(double input1, double input2, double expected) {
         return failures;
     }
 
+    static int testStrictVsNonstrictPowCase(double input1, double input2) {
+        double smResult = StrictMath.pow(input1, input2);
+        double mResult = Math.pow(input1, input2);
+        int failures = 0;
+        if (Double.isNaN(smResult)) {
+            if (!Double.isNaN(mResult)) failures = 1;
```

Comment:
Pretty sure the Tests.testUlpDiff call will handle two NaNs as well as NaN/non-NaN argument.

Reply:
Looks like that is true.